src/com.frostycoolslug.goxlr-utility.sdPlugin/app/display-volume.js

```diff
@@ -114,7 +115,44 @@ class VolumeMonitor {
 
         let percent = Math.round(((volume / 255) * 100));
 
-        let svg = `<svg height="100" width="100"><text fill="#fff" font-family="Monospace" font-size="26px" x="50&#37;" y="50&#37;" dominant-baseline="middle" text-anchor="middle">${percent}&#37;</text></svg>`;
+        let muteStatus = '';
+        let muteTarget = '';
+        for (const [key, value] of Object.entries(faders)) {
+            if (value.channel === this.channel_a) {
```

Comment:
This may cause problems because it doesn't correctly account for submixes, quick overview.

Mix A has more available channels than Mix B, so they've both had to be defined as separate drop downs in the property inspector. When switching between mixes it will simply hide one list and show the other list, due to this both values are passed into the action when configuring, and the selected channel is sorted out here when displaying, this means that A and B could be completely different channels.

There's some code above which determines whether to use channel A or channel B depending on `this.mix` and various 'available functionality' checks, but generally speaking you should be able to:

`let channel = (this.mix === "A") ? this.channel_a : this.channel_b`

To get the appropriate channel to check.

Reply:
I'll have to take a bit of time to review this one, thanks for the info